cpp_utils/BLEServiceMap.cpp

```diff
@@ -120,9 +124,17 @@ BLEService* BLEServiceMap::getNext() {
  * @brief Removes service from maps.
  * @return N/A.
  */
-void BLEServiceMap::removeService(BLEService* service) {
+void BLEServiceMap::removeService(BLEService *service){
```

Comment:
Please refrain from applying auto-formatting to the whole document when you only add\edit couple of lines.
Or, even better, set your IDE rules to conform to project's.